src/main/java/com/ihren/task2/exercise3/constant/StringUtils.java

```diff
@@ -0,0 +1,8 @@
+package com.ihren.task2.exercise3.constant;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+public class StringUtils {
```

Comment:
The constants class should be called 'Constants'. utils classes should be in util class and should contain logic